scripts/can_bundle.py

```diff
@@ -68,15 +68,21 @@ def fetch_bundle(source: str, bundle_uuid: str, bundle_version: str) -> Bundle:
     for catalog in config.catalogs:
         plugin = plugin_for(catalog)
         sources = set(map(str, plugin.sources))
-        if source in sources:
-            source = plugin.resolve_source(source)
-            fqid = SourcedBundleFQID(source=source,
-                                     uuid=bundle_uuid,
-                                     version=bundle_version)
-            bundle = plugin.fetch_bundle(fqid)
-            logger.info('Fetched bundle %r version %r from catalog %r.',
-                        fqid.uuid, fqid.version, catalog)
-            return bundle
+        try:
+            parsed_source = plugin.resolve_source(source)
+        except Exception:
+            raise
+        else:
+            for configured_source in sources:
```

Comment:
```suggestion
        parsed_source = plugin.resolve_source(source)
        for configured_source in sources:
```
If the exception is being caught just to be raised again, why not cut out the middle-man?
Or was `raise` intended to be `pass`?

Reply:
good point, in the sketchup we did with Hannes we wrote this as a `except: pass` But it seems silly to continue to work with a source that is passed in but does not parse correctly.